### ISSUE TITLE
Disable expected statement, but got empty on event handler

### ISSUE THREAD

**Issue by donnysim** [vuejs/eslint-plugin-vue#388]
**Tell us about your environment**

* **ESLint Version:** 4.17.0
* **eslint-plugin-vue Version:** 4.2.2
* **Node Version:** 8.7.0

**Please show your full configuration:**

```
module.exports = {
    parserOptions: {
        parser: 'babel-eslint',
        ecmaVersion: 6,
        sourceType: 'module',
        ecmaFeatures: {
            experimentalObjectRestSpread: true,
        },
    },
    env: {
        browser: true,
    },
    extends: [
        'plugin:vue/base',
    ],
    rules: {
        'vue/no-parsing-error': [2, {
            'missing-attribute-value': false,
        }],
    },
};
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
    <div @click.stop="" @mousedown.stop="">
    </div>
</template>
```

**What did you expect to happen?**
It would not complain about `@click` not having a value.

**What actually happened? Please include the actual, raw output from ESLint.**
It throws error `Expected to be statements, but got empty`. Maybe have a separate config that allows events to have an empty handler?

This is a valid and often encountered use case.

**Comment by mysticatea** [comment]
Thank you for the report.

I confirmed it. This is a bug in `vue-eslint-parser`.

**Comment by mysticatea** [comment]
I published `vue-eslint-parser@2.0.3` that should fix this issue.
Please try re-install.

**Comment by michalsnik** [comment]
Looks like this change breaks tests on fresh install. CircleCI seem to cache node_modules, but in fact they should fail now.

We expect following case to be invalid:
```
{
      filename: 'test.vue',
      code: '<template><div v-show="">hello</div></template>',
      errors: [
        { message: 'Parsing error: Expected to be an expression, but got empty.', column: 24 }
      ]
    },
```
But now it won't throw a parsing error. Is it desired @mysticatea ?

**Comment by mysticatea** [comment]
Yes, that test case is invalid and it's the bug I fixed. Vue.js seems to ignore empty attribute values without warnings: https://jsfiddle.net/5w9eyncg/
